src/site/xdoc/server/config-blob.xml

```diff
@@ -20,84 +20,31 @@
-                  <dd>Should be tmpauth</dd>
-
-                  <dt><strong>objectstorage.swift.endpoint</strong></dt>
-                  <dd>Swift authentication endpoint. Should be http://swift-host-address:8080/auth/v1.0 </dd>
-
-                  <dt><strong>objectstorage.swift.credentials</strong></dt>
-                  <dd>Should be testing</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.username</strong></dt>
-                  <dd>Should be tester</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.tenantname</strong></dt>
-                  <dd>Should be test</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
-                  <dd>Should be X-Storage-Pass</dd>
-
-                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
-                  <dd>Should be X-Storage-User</dd>
-              </dl>
-          </subsection>
-      </subsection>
-
       <subsection name="Blob Export Configuration">
           <p>
-              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-export-api/src/main/java/org/apache/james/blob/export/api/BlobExportMechanism.java">BlobExportMechanism</a>
-              is a component used to export blob content to a target email address. It requires to has a BlobId, an string of explanation, and target mail address in order to perform exporting.
+              Configuration for exporting blob content
           </p>
           <dl>
               <dt><strong>blob.export.implementation</strong></dt>
-              <dd>localFile: Local File Exporting Mechanism. Default: localFile</dd>
+              <dd>localFile: Local File Exporting Mechanism (explained below). Default: localFile</dd>
           </dl>
 
           <subsection name="Local File Blob Export Configuration">
               <p>
-                  This mechanism export blob data to a local zip file, then send an emails contains the absolute path of that file to the target mail address
+                  This mechanism export blob data to a local zip file, then send an email containing the absolute path of that file to the target mail address.
+                  Note: that absolute file path is the file location on James server. Therefore, if there are two or more James servers connected, it should not be considered an option.
               </p>
               <dl>
                   <dt><strong>blob.export.localFile.directory</strong></dt>
-                  <dd>The directory URL to store exported blob data in files, and the URL follows James <a href="https://github.com/apache/james-project/server/container/filesystem-api/src/main/java/org/apache/james/filesystem/api/FileSystem.java">FileSystem</a> scheme</dd>
+                  <dd>The directory URL to store exported blob data in files, and the URL following James File System scheme. Default: file://var/blobExporting</dd>
```

Comment:
is the scheme documented somewhere?

Reply:
I can't find a better document than the API doc [here](http://james.apache.org/server/3/apidocs/org/apache/james/filesystem/api/FileSystem.html)